docs/v3.x/getting-started/nuxt-js.md

```diff
@@ -0,0 +1,755 @@
+
+  <form id="form" v-on:submit="handleSubmit" v-else>
+    <h3>Restaurants</h3>
+    <br>
+
+    <label for="name">Name</label>
+    <input id="name" v-model="name" type="text" name="name">
+
+    <label for="description">Description</label>
+    <input id="description" v-model="description" type="text" name="description">
+    <div>
+      <br />
+      <b>Select categories</b>
+      <br>
+      <div v-for="category in allCategories" :key="category.id">
+        <label>{{ category.name }}</label>
+        <input
+          type="checkbox"
+          :value="category.id"
+          v-model="categories"
+          name="categories"
+          :id="category.id"
+        />
+      </div>
+    </div>
+    <br>
+
+    <input type="submit" value="Submit">
+  </form>
+
+</div>
+</template>
+
+<script>
+export default {
+  name: 'App',
+  data() {
+    return {
+      allCategories: [],
+      name: '',
+      description: '',
+      categories: [],
+      error: null
+    }
+  },
+  async mounted() {
+    try {
+      this.allCategories = await this.$strapi.find('categories')
+    } catch (error) {
+      this.error = error
+    }
+  },
+  methods: {
+    handleSubmit: async function(e) {
+      try {
+        await this.$strapi.create('restaurants', {
+          name: this.name,
+          description: this.description,
+          categories: this.categories
+        })
+      } catch (error) {
+        this.error = error
+      }
+
+      e.preventDefault();
+    }
+  }
+}
+</script>
+```
+
+:::
+
+::: tab axios
+
+`./pages/index.vue`
+
+```js
+<template>
+<div class="container">
+  <div v-if="error">
+    {{ error }}
+  </div>
+
+  <form id="form" v-on:submit="handleSubmit" v-else>
+    <h3>Restaurants</h3>
+    <br>
+
+    <label for="name">Name</label>
+    <input id="name" v-model="name" type="text" name="name">
+
+    <label for="description">Description</label>
+    <input id="description" v-model="description" type="text" name="description">
+    <div>
+      <br />
+      <b>Select categories</b>
+      <br>
+      <div v-for="category in allCategories" :key="category.id">
+        <label>{{ category.name }}</label>
+        <input
+          type="checkbox"
+          :value="category.id"
+          v-model="categories"
+          name="categories"
+          :id="category.id"
+        />
+      </div>
+    </div>
+    <br>
+
+    <input type="submit" value="Submit">
+  </form>
+
+</div>
+</template>
+
+<script>
+import axios from 'axios'
+
+export default {
+  name: 'App',
+  data() {
+    return {
+      allCategories: [],
+      name: '',
+      description: '',
+      categories: [],
+      error: null
+    }
+  },
+  mounted() {
+    axios.get('http://localhost:1337/categories')
+      .then(response => {
+        this.allCategories = response.data;
+      })
+      .catch(error => {
+        this.error = error;
+      })
+  },
+  methods: {
+    handleSubmit: function(e) {
+      axios.post('http://localhost:1337/restaurants', {
```

Comment:
Why sometimes are you using async functions and sometime promises? Can you make it similar so it is easier for the users to understand?

In fact the only code that should differ in your example is when you use fetch and axios:

```
// With fetch
try {
  const response = await fetch('url', { method: 'POST', body: modifiedData }).then(checkStatus).then(parseJSON);
} catch(err) {
  console.log(err)
} 

// With axios
try {
  const response = await axios.post('url', modifiedData).then(checkStatus).then(parseJSON);
} catch(err) {
  console.log(err)
} 
```